extensions/extension.go

```diff
@@ -0,0 +1,34 @@
+package extensions
+
+import (
+	"math"
+
+	"grol.io/grol/object"
+)
+
+func init() {
+	Init()
+}
+
+func Init() {
```

Comment:
Do you really need an Init, it could have been in init

Please add a comment if needed

Reply:
Unless you do this

https://github.com/grol-io/grol/pull/96#discussion_r1702804357

Reply:
I usually have Init() callable by unit test (see token.go for a good example) but yes

Reply:
done in https://github.com/grol-io/grol/pull/96/commits/2fc25bc43e98d1d01ae13225bc8a61396922a857 ptal